pkg/core/mpt/extension.go

```diff
@@ -11,8 +11,14 @@ import (
 	"github.com/nspcc-dev/neo-go/pkg/util"
 )
 
-// MaxKeyLength is the max length of the extension node key.
-const MaxKeyLength = (storage.MaxStorageKeyLen + 4) * 2
+const (
+	// maxPathLength is the max length of the extension node key.
+	maxPathLength = (storage.MaxStorageKeyLen + 4) * 2
+
+	// maxKeyLength is the max length of the key to put in trie
+	// before transforming to nibbles.
+	maxKeyLength = maxPathLength / 2
```

Comment:
I think it's OK to have it exported for documentation purposes, because there is this limit.